src/WebUI/Services/ServiceDetails.tsx

```diff
@@ -197,9 +197,16 @@ export class ServiceDetails extends BaseComponent<IServiceDetailsProperties, ISe
 
     private _getAssociatedPods(): JSX.Element | null {
         if (this.state.pods.length === 0) {
-            return KubeZeroData._getDefaultZeroData("https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/",
-                Resources.LearnMoreText, Resources.NoPodsForSvcText,
-                Resources.LinkSvcToPodsText, Resources.AssociatedPodsText);
+            const zeroDataProps: IKubeZeroDataProps = {
+                hyperLink: "https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/",
```

Comment:
please keep the hyperlink in a const file, also please send an email to PM to get the links